src/cpp/core/Error.cpp

```diff
@@ -155,6 +156,20 @@ bool Error::isError() const
       return false ;
 }
 
+bool Error::isShutdownError(const boost::system::error_code& ec)
```

Comment:
That's right -- should it be part of the `core` or `http` namespace? (Seems like `core` to me just because the errors are not necessarily scoped just to `http` server implementations)